pkg/getty/usr/bin/rungetty.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/sh
+set -x
+
+infinite_loop() {
+	while true; do
+		$@
+	done
+}
+
+# run getty on all known consoles - except those already in inittab
+start_getty() {
+	tty=${1%,*}
+	speed=${1#*,}
+	inittab="$2"
+	securetty="$3"
+	line=
+	term="linux"
+	[ "$speed" = "$1" ] && speed=115200
+
+	case "$tty" in
+	ttyS*|ttyAMA*|ttyUSB*|ttyMFD*)
+		line="-L"
+		term="vt100"
+		;;
+	tty?)
+		line=""
+		speed="38400"
+		term=""
+		;;
+	esac
+
+	# are we secure or insecure?
+	loginargs=
+	if [ "$INSECURE" == "true" ]; then
+		loginargs="-n -l /bin/sh"
+	fi
+
+	# skip consoles already in inittab
+	#grep -q "^$tty:" "$inittab" && return
+
+	#echo "$tty::respawn:/sbin/getty -n -l /bin/sh $line $speed $tty $term" >> "$inittab"
+	if ! grep -q -w "$tty" "$securetty"; then
+		echo "$tty" >> "$securetty"
+	fi
+	# respawn forever
+	infinite_loop setsid -w /sbin/getty $loginargs $line $speed $tty $term &
+}
+
+# check if we have /etc/getty.shadow
+ROOTSHADOW=/proc/1/root/etc/getty.shadow
```

Comment:
If you can run `ctr` you are controlling `containerd` that runs in the host namespace and will give you root access and allow you to bind mount from root, whichever namespace you are in. Same issue with Docker (will fix this one day...).

Yes I guess you can bindmount `/etc`